src/main/java/com/github/manolo8/darkbot/core/objects/facades/SlotBarsProxy.java

```diff
@@ -12,9 +13,22 @@
     //public SlotBar premiumBar    = new SlotBar(); //104
     //public SlotBar proActionBar  = new SlotBar(); //112
 
+    private Main main;
```

Comment:
Did you even test this change? this main is never initialized.

Reply:
Tested, and failed as i wrote in commit name. Can you tell me how should i initialize or what to do?